IDEAS/Buildings/Components/Shading/BuildingShade.mo

```diff
@@ -7,15 +7,23 @@ model BuildingShade
   parameter Modelica.Units.SI.Length L(min=0)
     "Distance to object perpendicular to window"
     annotation (Dialog(group="Dimensions (see illustration in documentation)"));
+  parameter Modelica.Units.SI.Length LOv(min=0) = 0
```

Comment:
I would unify the parameter naming of this model and the `Overhang` model. So either:

1. `dep` (as used in the Overhang model)
2. `LOve` (use three characters) and apply this change to the Overhang model as well